web/src/components/Ticket/Ticket.js

```diff
@@ -69,30 +72,46 @@ const Ticket = ({ ticket }) => {
               <td>{ticket.title}</td>
             </tr>
             <tr>
-              <th>User id</th>
-              <td>{ticket.userId}</td>
+              <th>Impact</th>
+              <td>{propercase(ticket.impact)}</td>
+            </tr>
+            <tr>
+              <th>Urgency</th>
+              <td>{propercase(ticket.urgency)}</td>
+            </tr>
+            <tr>
+              <th>Priority</th>
+              <td>{propercase(ticket.priority)}</td>
+            </tr>
+            <tr>
+              <th>Title</th>
+              <td>{ticket.title}</td>
+            </tr>
+            <tr>
+              <th>Assigned To</th>
+              <td>{ticket.User.name}</td>
```

Comment:
Add Impact, Urgency, and Priority to the mix.  Show who the "User" is by name and not number.